cluster-autoscaler/hack/update-codegen.sh

```diff
@@ -27,7 +27,7 @@ GO_CMD=${1:-go}
 CURRENT_DIR=$(dirname "${BASH_SOURCE[0]}")
 REPO_ROOT="$(git rev-parse --show-toplevel)"
 CODEGEN_PKG=$($GO_CMD list -m -mod=readonly -f "{{.Dir}}" k8s.io/code-generator)
-cd "${CURRENT_DIR}/.."
+cd "${CURRENT_DIR}/../apis"
```

Comment:
Without this change `./hack/update-codegen.sh` fails with:

```
F1205 11:51:07.357544 1794877 main.go:68] Error: failed making a parser: error(s) in "k8s.io/autoscaler/cluster-autoscaler/apis/provisioningrequest/autoscaling.x-k8s.io/v1":
-: main module (k8s.io/autoscaler/cluster-autoscaler) does not contain package k8s.io/autoscaler/cluster-autoscaler/apis/provisioningrequest/autoscaling.x-k8s.io/v1
error(s) in "k8s.io/autoscaler/cluster-autoscaler/apis/provisioningrequest/autoscaling.x-k8s.io/v1beta1":
-: main module (k8s.io/autoscaler/cluster-autoscaler) does not contain package k8s.io/autoscaler/cluster-autoscaler/apis/provisioningrequest/autoscaling.x-k8s.io/v1beta1
```

Looks like we may not have ran this script in awhile..